WORKSPACE

```diff
@@ -52,21 +52,21 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 http_archive(
     name = "io_bazel_rules_go",
     urls = [
-        "https://storage.googleapis.com/bazel-mirror/github.com/bazelbuild/rules_go/releases/download/v0.20.3/rules_go-v0.20.3.tar.gz",
-        "https://github.com/bazelbuild/rules_go/releases/download/v0.20.3/rules_go-v0.20.3.tar.gz",
+        "https://storage.googleapis.com/bazel-mirror/github.com/bazelbuild/rules_go/releases/download/0.17.0/rules_go-0.17.0.tar.gz",
```

Comment:
Lowering the `io_bazel_rules_go` and `bazel_gazelle` versions had issues to build other targets in the workspace (e.g., pip wheels, and cc). let's keep them as is for now?  Please also see the background in the metadata_store/BUILD comment below.